samples/src/Microsoft.Azure.IIoT.App/src/Models/PublisherInfoRequested.cs

```diff
@@ -9,43 +9,32 @@ namespace Microsoft.Azure.IIoT.App.Models {
     public class PublisherInfoRequested {
 
         public PublisherInfoRequested(PublisherInfo publisher) {
-            _requestedMaxWorkers = publisher?.PublisherModel?.Configuration?.MaxWorkers;
-            _requestedHeartbeatInterval = publisher?.PublisherModel?.Configuration?.HeartbeatInterval;
-            _requestedJobCheckInterval = publisher?.PublisherModel?.Configuration?.JobCheckInterval;
-        }
+            RequestedMaxWorkers = publisher?.PublisherModel?.Configuration?.MaxWorkers.ToString();
+
+            TimeSpan? heartbeatInterval = publisher?.PublisherModel?.Configuration?.HeartbeatInterval;
+            if (heartbeatInterval.HasValue && heartbeatInterval >= TimeSpan.Zero) {
+                RequestedHeartbeatInterval = heartbeatInterval.Value.TotalSeconds.ToString();
+            }
 
-        private int? _requestedMaxWorkers;
-        private TimeSpan? _requestedHeartbeatInterval;
-        private TimeSpan? _requestedJobCheckInterval;
+            TimeSpan? jobCheckInterval = publisher?.PublisherModel?.Configuration?.JobCheckInterval;
+            if (jobCheckInterval.HasValue && jobCheckInterval >= TimeSpan.Zero) {
```

Comment:
jobCheckInterval could be null

Reply:
I must disagree (a nullable type is a struct which will be boxed as null if the HasValue property is false), but it would be better to use jobCheckInterval.Value property instead of an implicit cast when comparing to TimeSpan.Zero. I will fix that.

Reply:
Right, I oversaw that, txs